google/cloud/spanner_v1/_helpers.py

```diff
@@ -17,9 +17,12 @@
 import datetime
 import decimal
 import math
+import base64
 
 from google.protobuf.struct_pb2 import ListValue
 from google.protobuf.struct_pb2 import Value
+from google.protobuf.message import Message
```

Comment:
You need to expose these types in the init file for customers to be able to use this data type

Reply:
I guess init file has only the types that exist in python-spanner client library. The Message is a type from google.protobuf.message and I don't think this should be added.
Additionally, customers do not use this type directly. They use the references of proto message and enum from the auto generated code ([singer_pb2.py](https://github.com/googleapis/python-spanner/pull/909/files#diff-9ba15bb21b15c25ea6a17fb3c6ec868cfb26dcbeb403f50b21642f44433d2684)). For ex, the sample for this would look like [insert sample](https://github.com/googleapis/python-spanner/pull/909/files#diff-62f4ce45093b6508acb281948c5d4ab7121da9db4d7ad1eb1dd69d58c42a0f1eR2521) where the customer does not use Message directly.

Reply:
Okay got it.